public/client.html

```diff
@@ -36,6 +36,6 @@
 </div>
 
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="https://www.youtube.com/iframe_api"></script>

```

Comment:
The order of the imports here was intended. We had cases in which the callback defined here:  https://github.com/Yavor-Ivanov/community-driven-radio/blob/4f0c6bfcd393d6a9cacfc68bde1868e2e0d02d6e/public/badass-client.js#L164 for the YouTube iframe API, was sometimes not called when  `https://www.youtube.com/iframe_api` loads before our JS files.

Did you have any experiences with this, or it was moved for aesthetics?